packages/wallet/frontend/src/lib/api/user.ts

```diff
@@ -139,6 +139,9 @@ export const verifyIdentitySchema = z
 export const forgotPasswordSchema = z.object({
   email: z.string().email({ message: 'Email is required' })
 })
+export const resendVerificationEmailSchema = z.object({
```

Comment:
move the schema to shared lib and use the same schema in both frontend and backend

Reply:
as forgot password has the same schema use one generic schema for both cases